packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -71,25 +63,11 @@ export default function CollabSidebar() {
 
 		return {
 			postId: _postId,
-			postStatus: getEditedPostAttribute( 'status' ),
-			threads: data ?? EMPTY_ARRAY,
-		};
-	}, [] );
-
-	const { clientId, blockCommentId } = useSelect( ( select ) => {
-		const { getBlockAttributes, getSelectedBlockClientId } =
-			select( blockEditorStore );
-		const _clientId = getSelectedBlockClientId();
-
-		return {
-			clientId: _clientId,
-			blockCommentId: _clientId
-				? getBlockAttributes( _clientId )?.blockCommentId
-				: null,
+			threads: data,
 		};
 	}, [] );
 
-	// Get the dispatch functions to save the comment and update the block attributes.
+	const { getSelectedBlockClientId } = useSelect( blockEditorStore );
```

Comment:
The `clientId` is now only used in event callback; we can use the static getter.